services/cloudfront.py

```diff
@@ -2,40 +2,137 @@
 import boto3
 
 
-# client = boto3.client("")
+client = boto3.client("cloudfront")
 
 
 def cloudfront_accesslogs_enabled():
+    compliant_resources = []
+    non_compliant_resources = []
+    distributions = client.list_distributions()["DistributionList"]["Items"]
+
+    for distribution in distributions:
+        distribution = client.get_distribution(Id=distribution["Id"])["Distribution"]
+        if (
+            "Logging" in distribution["DistributionConfig"]
+            and distribution["DistributionConfig"]["Logging"]["Enabled"] == True
+        ):
+            compliant_resources.append(distribution["ARN"])
+        else:
+            non_compliant_resources.append(distribution["ARN"])
+
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resources,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def cloudfront_associated_with_waf():
+    compliant_resources = []
+    non_compliant_resources = []
+    distributions = client.list_distributions()["DistributionList"]["Items"]
+
+    for distribution in distributions:
+        distribution = client.get_distribution(Id=distribution["Id"])["Distribution"]
+
+        if "WebACLId" in distribution["DistributionConfig"] and distribution["DistributionConfig"]["WebACLId"] != "":
```

Comment:
`get_distribution`을 한번 더 호출하지 않아도 됩니다.
```suggestion
        if "WebACLId" in distribution and distribution["WebACLId"] != "":
```